src/ui/actions/reactDevTools.ts

```diff
@@ -57,8 +20,6 @@ export function addAnnotations(annotations: Annotation[]): AddAnnotationsAction
   return { type: "add_annotations", annotations };
 }
 
-export function setProtocolCheckFailed(): UIThunkAction {
-  return async dispatch => {
-    dispatch({ type: "set_protocol_fail" });
-  };
+export function setProtocolCheckFailed(): SetProtocolCheckFailedAction {
```

Comment:
:speech_balloon: This action creator really didn't need to be a thunk anyway, since it's just dispatching a single basic action.